sk-csharp-chatgpt-plugin/azure-function/Extensions/ImportPromptsFromDirectory.cs

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.Extensions.Logging;
+using Microsoft.SemanticKernel;
+using Microsoft.SemanticKernel.Diagnostics;
+using Microsoft.SemanticKernel.SkillDefinition;
+using Microsoft.SemanticKernel.SemanticFunctions;
+using Microsoft.SemanticKernel.Text;
+
+namespace AIPlugins.AzureFunctions.Extensions;
+
+public static class ImportPromptsFromDirectoryExtensions
```

Comment:
`KernelExtensions` is a better name for the file and the class

Reply:
Fixed